extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -692,21 +716,17 @@ export class AmpAnalytics extends AMP.BaseElement {
     const dynamicBindings = {};
     const resourceTimingSpec = trigger['resourceTimingSpec'];
     if (resourceTimingSpec) {
-      const on = trigger['on'];
-      if (on == 'ini-load') {
+      const lastTime = resourceTimingSpec['responseAfter'] || 0;
+      if (!resourceTimingSpec['done'] ||
+          lastTime < RESOURCE_TIMING_MAX_REPORT_TIME_MSEC) {
+        resourceTimingSpec['responseAfter'] =
```

Comment:
This wasn't a bug because `lastTime` was explicitly passed to `serializeResourceTimingEntries`.

However, I admit the code is confusing. I moved this logic to resource-timing.js so all relevant logic is in one place. I added test coverage. PTAL